src/store/index.ts

```diff
@@ -0,0 +1,9 @@
+import Vue from 'vue';
+import VueCompositionAPI, { ref } from '@vue/composition-api';
+import { GeoJsonShape } from './types';
+
+Vue.use(VueCompositionAPI);
```

Comment:
I don't believe you're supposed to call `Vue.use(VueCompositionAPI)` multiple times within your code, and it's already called initially in `main.ts`, so you should be able to just remove it here. If you do so, there's a chance you'll run into the following error
```
[vue-composition-api] must call Vue.use(VueCompositionAPI) before using any function.
```

If that occurs, it can be solved by creating a new plugin file (`plugins/composition.ts` or something), placing the `Vue.use` in there, and importing that plugin in `main.ts` like so:

```typescript
// plugins/composition.ts
import Vue from 'vue';
import VueCompositionAPI from '@vue/composition-api';

Vue.use(VueCompositionAPI);
```
```typescript
// main.ts
import '@/plugins/composition';
```

If you do so, just make sure that it's imported _before_ any of your own code is imported, to ensure that if any of your imports attempt to use Composition API, all goes smoothly.

Reply:
Yes, I did indeed get that error so I will create the plugin. Thank you for this solution.